src/interface_test.jl

```diff
@@ -11,24 +11,28 @@ that the behavior of [`logdensityof(d)`](@ref),
 
 Also tests if `logfuncdensity(logdensityof(d))` returns
 a density equivalent to `d` in respect to the functions above.
+
+The results of `logdensityof(d, x)` and `densityof(d, x)` are compared to
+`ref_logd_at_x` and `exp(ref_logd_at_x)` using `compare`. `kwargs...` are
+forwarded to `compare`.
 """
-function test_density_interface(d, x, ref_logd_at_x)
+function test_density_interface(d, x, ref_logd_at_x; compare=isapprox, kwargs...)
```

Comment:
Do we actually want to not use `isapprox` in some cases? Otherwise we could just add `kwargs` and forward them to `isapprox`.

Reply:
Hm, true, the results are, by definition, real numbers. Done.